scripts/ldap/setup_ldap.sh

```diff
@@ -182,6 +180,10 @@ ldapmodify -D "cn=admin,cn=config" -w $ROOTDNPASSWORD -a -f $INSTALLDIR/ppolicyo
 
 ldapmodify -x -a -H ldapi:/// -D cn=admin,dc=seagate,dc=com -w "$ROOTDNPASSWORD" -f "$INSTALLDIR"/ppolicy-default.ldif || /bin/true
 
+# add S3 schema
+ldapadd -x -D "cn=admin,cn=config" -w $ROOTDNPASSWORD -f $INSTALLDIR/cn\=\{2\}s3user.ldif -H ldapi:///
```

Comment:
ldapi might work local only, can you check this

Reply:
This is not the file which is getting used.